alphagov/paas-cf
# WIP: [#156877256] Setup Datadog monitors for DNS resolution

## What

This adds 2 monitors around DNS resolution on each VM: 1 checks that DNS
resolution works, the 2nd checks that it resolves in a timely manner.
    
These are being added because we had a recent issue that we eventally
tracked down to DNS resolution on one VM being flakey. In that instance
restarting the VM fixed it. Having this check will enable us to see this
sort of cause much more quickly.

## How to review

* Deploy alphagov/paas-bootstrap#160
* Deploy from this branch (it includes a TMP commit to enable datadog)
* Check that the monitors are configured in Datadog, and that they work.
* Check that the thresholds etc I've chosen make sense.

## Who can review

Not me
